uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseGridWidget.java

```diff
@@ -854,4 +856,10 @@ public boolean startEditingCell(final int uiRowIndex,
     public boolean startEditingCell(final Point2D rp) {
```

Comment:
@jomarko 
I know it depends on already existing implementation, but it would be much better/cleaner if all those operation could accept the same parameters (rowIndex, columnIndex). I think it would just need an overloaded method inside the cellSelectionManager.startEditingCell

Reply:
No. We have the overloaded methods as it's possible to start editing a cell programmatically (row, column index) or by User interaction (mouse location). In the former case a "popover editor" is shown relative to the top-left of the cell; when invoked by User interaction the "popover editor" is at the mouse location.